nevergrad/optimization/test_optimizerlib.py

```diff
@@ -494,3 +494,31 @@ def test_ngo_split_optimizer(name: str, expected: tp.List[str]) -> None:
     opt = optlib.ConfSplitOptimizer(multivariate_optimizer=Opt)(param, budget=1000)
     names = [o.optim.name for o in opt.optims]  # type: ignore
     assert names == expected
+
```

Comment:
```suggestion


```

As per pep8, please always keep 2 lines between independent base level functions (aka not methods)